Plan/api/src/main/java/com/djrapitops/plan/delivery/web/resolver/request/Request.java

```diff
@@ -43,16 +44,18 @@
      * @param query   Request parameters ?param=value etc
      * @param user    Web user doing the request (if authenticated)
      * @param headers Request headers https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers
+     * @param body    Raw body as bytes, if present
      */
-    public Request(String method, URIPath path, URIQuery query, WebUser user, Map<String, String> headers) {
+    public Request(String method, URIPath path, URIQuery query, WebUser user, Map<String, String> headers, byte[] body) {
         this.method = method;
         this.path = path;
         this.query = query;
         this.user = user;
         this.headers = headers;
+        this.body = body;
     }
 
-    // Special constructor that figures out URIPath and URIQuery from "/path/and?query=params"
+    // Special constructor that figures out URIPath and URIQuery from "/path/and?query=params" and has no form body
```

Comment:
This should probably be a javadoc comment
```suggestion
    /**
     * Special constructor that figures out URIPath and URIQuery from "/path/and?query=params" and has no form body.
     */
```